src/index.html

```diff
@@ -16,17 +16,38 @@
       href="style.css"
     />
   </head>
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form class="form form__1">
+      <div class="input__wrapper">
+        <img
+          src="images/Search.svg"
+          alt="search-logo"
+          class="input__icon input__icon__1"
```

Comment:
The `alt` attribute should describe the image content. 'search-logo' is not descriptive of the image content. It should describe the action or purpose of the image, like 'Search icon'.